e2e/windows/cleanup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# delete blobs created 3 days ago
+EXPIRATION_IN_HOURS=72
+(( expirationInSecs = ${EXPIRATION_IN_HOURS} * 60 * 60 ))
+(( deadline=$(date +%s)-${expirationInSecs%.*} ))
+dateOfdeadline=$(date -d @${deadline} +"%Y-%m-%dT%H:%M:%S+00:00")
+
+# two containers need to be cleaned up now
+CONTAINER_LIST=("cselogs" "csepackages")
```

Comment:
could we support deleting the containers created by @wanqingfu's PR?

Reply:
Not sure why we need to delete the containers. We use one container for one kubernetes version. Every time the resource group is deleted automatically, we will also clean up the files in the container before we upload new files from new created resource group.

Reply:
I think that we can add this after Qingfu's PR is merged.